### PR TITLE
Added Check Script

### PR DESCRIPTION
**Pull request by himanshukm2301**

### CODE REVIEW

File: .github/workflows/percy.yml [hunk 1]
```diff
@@ -33,3 +33,20 @@ jobs:
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run build --if-present
+    - run: npm test
```
Comment (swarajpure) [inline .github/workflows/percy.yml:52]: We just need `npm install` and `npm run check`
We don't need `build` and `test`

Comment (himanshukm2301) [reply .github/workflows/percy.yml]: Updated

**Review by swarajpure — CHANGES_REQUESTED** [review]
Great! Thanks 🎉 

Just small changes:
image

These don't make much sense, right? Can you see where they're coming from and how to fix them? First one is okay, second one should say something like "Checking formatting"

**Comment by himanshukm2301** [comment]
Sure, I'll look into it!

**Review by swarajpure — APPROVED** [review]
Looks good to me!
Thank you! 🎉

**Pull request merged**